fork-off/README.md

```diff
@@ -31,13 +33,15 @@ The tool will perform the following actions, in this order:
 2. Dump the state to a json file. You can provide a path via `--snapshot-path`.
 3. Read the state from the snapshot json file. This is because steps 1. and 2. can be omitted by running with `--use-snapshot-file` -- see example below.
 4. Read the chainspec provided via `--initial-spec-path` you should pass here the one generated via `the bootstrap-chain` command, so `--initial-spec-path=chainspec.json` if it is in the same directory.
-5. Replace the genesis state in the chainspec by the one from the snapshot WITH THE EXCEPTION of states of paths provided via a comma separated list using `--storage_keep_state`. The default setting is `--storage_keep_state=Aura,Aleph,Balances,Sudo,Staking,Session,Elections,System.Account` and it's likely you don't want to change it.
-6. The final, new chainspec is saved to the path provided via `--combined-spec-path`.
+5. Replace the genesis state in the chainspec by the one from the snapshot WITH THE EXCEPTION of states of paths provided via a comma separated list using `--storage_keep_state`. The default setting is `--storage_keep_state=Aura,Aleph,Sudo,Staking,Session,Elections` and it's likely you don't want to change it.
+6. If you have passed `--accounts_path` pointing to a file with a configuration for some accounts, then it will be written into chainspec (`System.Account` map). For an example see `AccountInfo-Template.json` file.
+7. Alternatively to `--accounts_path` you can just pass `--balances` flag with which you can specify initial free balances for some accounts. Similarly, it will be saved to `System.Account` map.
```

Comment:
Can we elaborate on some details on both `--accounts_path` and `--balances` ie what type of account we pass (ie SS58 or hex), and what format exactly have `--balances`. This is to enrich the user experience. :)